.example.env

```diff
@@ -42,4 +42,7 @@ NEXT_PUBLIC_POSTHOG_HOST=https://us.i.posthog.com
 
 
 # Prisma
-DATABASE_URL="postgresql://user:password*@domain:5432/intown_dev"
\ No newline at end of file
+DATABASE_URL="postgresql://user:password*@domain:5432/intown_dev"
+
+NEXTAUTH_URL=http://localhost:3000
+NEXTAUTH_SECRET=[generate with `openssl rand -base64 32`]
```

Comment:
So, if these are new ENV vars, go into Render -> Projects -> Staging Env Var Group -> Add Env Vars

And add these new vars.

- NEXTAUTH_URL should be the url of the app in that env.
- NEXTAUTH_SECRET can be set to something random like you're doing here.